$MirOS: ports/print/cups/patches/patch-berkeley_lpr_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- berkeley/lpr.c.orig	Mon Mar  6 13:02:00 2006
+++ berkeley/lpr.c	Tue May  9 17:39:25 2006
@@ -299,7 +299,7 @@ main(int  argc,				/* I - Number of comm
 	      num_copies = atoi(argv[i]);
 	    }
 
-            sprintf(buffer, "%d", num_copies);
+            snprintf(buffer, sizeof(buffer), "%d", num_copies);
             num_options = cupsAddOption("copies", buffer, num_options, &options);
 	    break;
 
